rpm/osg-ca-certs.spec

```diff
@@ -12,7 +12,7 @@ URL:            http://repo.opensciencegrid.org/cadist/
 
 Source0:        https://github.com/opensciencegrid/osg-certificates/archive/v%{vtag}/osg-certificates-%{vtag}.tar.gz
 Source1:        https://dist.eugridpma.info/distribution/igtf/current/igtf-policy-installation-bundle-%{igtf_version}.tar.gz
-Source2:        https://github.com/cilogon/letsencrypt-certificates/archive/master/letsencrypt-certificates.tar.gz
+Source2:        https://github.com/brianhlin/letsencrypt-certificates/archive/7d89386f51e830a638798f3c6c5c0375be96d20c/letsencrypt-certificates.tar.gz
```

Comment:
Not that your change here breaks anything, but i believe everything except the basename (`letsencrypt-certificates.tar.gz`) is ignored / basically treated as documentation.  So the only important thing is that the `letsencrypt-certificates.tar.gz` part matches the filename in osg-ca-certs/upstream/letsencrypt.tarball.source in the svn packaging.

Reply:
Yup, that name matches what we've got in `upstream`. Scratch builds do work so I was able to get that far :)

Reply:
Right - i would expect that this change doesn't break anything; i was just noting that i don't think it's actually necessary either.

Reply:
Ah, sure. I figured the change is nice for clarity